src/App.tsx

```diff
@@ -1,26 +1,172 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import * as todoService from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './Components/TodoList';
+import { Footer } from './Components/Footer';
+import { Header } from './Components/Header';
+import cn from 'classnames';
+import { ErrorTypes } from './types/ErrorTypes';
+import { Loader } from './Components/Loader/Loader';
+import { Status } from './types/Status';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [listOfTodos, setListOfTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [selectedValue, setSelectedValue] = useState(Status.all);
+  const [errorMessage, setErrorMessage] = useState<ErrorTypes | null>(null);
+  const [query, setQuery] = useState('');
+  const [responding, setResponding] = useState(false);
```

Comment:
Consider changing the naming to `isResponding` to better describe that this is a boolean state